frontend/src/helper.js

```diff
@@ -94,6 +94,11 @@ export const toAmountString = (amount, currency) => {
   return accounting.formatMoney(amount, getCurrencyFormat(currency));
 };
 
+export const testMoneyInput = amount => {
```

Comment:
Remove the comment and rename the function so it's clear what the function does

Reply:
renamed to checkAmountMoneyPattern